test/arb/ComplexPoly-test.jl

```diff
@@ -7,8 +7,8 @@ coeff_types = [Int8, Int, UInt, BigInt,
                ZZRingElem, QQFieldElem]
 
 @testset "ComplexPolyRingElem.constructors" begin
-  S1 = PolyRing(CC)
-  S2 = PolyRing(CC)
+  S1 = polynomial_ring(CC; cached=false)
+  S2 = polynomial_ring(CC; cached=false)
```

Comment:
```suggestion
  S1 = polynomial_ring(CC; cached=false)[1]
  S2 = polynomial_ring(CC; cached=false)[1]
```
this needs such a change for all touched files

Reply:
indeed, done